### ISSUE TITLE
min max l/@n for witnesses

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-gen-html#557]
Ideally, we would also record gaps like

         <l n="1">erster</l>
         <l n="2">zweiter</l>
         <l n="4">vierter</l>
         <l n="5">fünfter Schröer-Vers</l>

as 1--2, 4--5

**Comment by gerritbruening** [comment]
That's correct. In many cases, mere min max would be misleading, though, because gaps may be quite large. @dietmarpravida, what do yo think?

**Comment by thvitt** [comment]
I’ve just pushed min–max with a link to that interval in the bar graph. 

To actually list intervals here, the bargraph data extraction would need to be refactored such that its collected info could be used from the XSLTs

**Comment by dietmarpravida** [comment]
Personally I should like to have gaps recorded. However, I guess there will be endless complications, especially with pencil drafts.

**Comment by dietmarpravida** [comment]
It would be much easier to use the informations in the gsa inventary.

**Comment by gerritbruening** [comment]
gsa inventary = http://dev.faustedition.net/bibliography#inventare_2_2
I would rather prefer not to use this information but our data.
